.buildkite/scripts/pipelines/pull_request/pipeline.ts

```diff
@@ -189,7 +191,47 @@ const uploadPipeline = (pipelineContent: string | object) => {
       ])) ||
       GITHUB_PR_LABELS.includes('ci:all-cypress-suites')
     ) {
-      pipeline.push(getPipeline('.buildkite/pipelines/pull_request/defend_workflows.yml'));
+      pipeline.push(
+        getPipeline('.buildkite/pipelines/pull_request/security_solution/defend_workflows.yml')
+      );
+    }
+
+    if (
+      (await doAnyChangesMatch([
+        /^package.json/,
+        /^packages\/kbn-securitysolution-.*/,
+        /^x-pack\/plugins\/alerting/,
+        /^x-pack\/plugins\/data_views\/common/,
+        /^x-pack\/plugins\/lists/,
+        /^x-pack\/plugins\/rule_registry\/common/,
+        /^x-pack\/plugins\/security_solution/,
+        /^x-pack\/plugins\/security_solution_ess/,
+        /^x-pack\/plugins\/security_solution_serverless/,
+        /^x-pack\/plugins\/task_manager/,
+        /^x-pack\/plugins\/timelines/,
+        /^x-pack\/plugins\/triggers_actions_ui\/public\/application\/sections\/action_connector_form/,
+        /^x-pack\/plugins\/triggers_actions_ui\/public\/application\/context\/actions_connectors_context\.tsx/,
+        /^x-pack\/plugins\/usage_collection\/public/,
+        /^x-pack\/test\/security_solution_cypress/,
```

Comment:
Do we also need to specify our FTRs here? Right now they are under `security_solution_api_integration` with only a few left in `detection_engine_api_integration`.

Reply:
Good question. As far as I know, the top priority right now is Cypress. We can do a follow-up PR to add conditional execution for the FTR.

Reply:
@yctercero I don't think we need to specify these folders here because our FTR tests run unconditionally in all PRs, as all other FTR tests and as configured here:

https://github.com/elastic/kibana/blob/1a0dcf60fc3f2a8d498ae68f92db53a8de7fd112/.buildkite/ftr_configs.yml#L461-L514